### PR TITLE
libreoffice: update to 7.2.0.1

### PR DESCRIPTION
**Pull request by Tatsh**
###### Type(s)

- [ ] bugfix
- [ ] enhancement
- [ ] security fix

###### Tested on

macOS 11.4 20F71 x86_64
Xcode 12.5.1 12E507

###### Verification 
Have you

- [x] followed our [Commit Message Guidelines](https://trac.macports.org/wiki/CommitMessages)?
- [x] squashed and [minimized your commits](https://guide.macports.org/#project.github)?
- [x] checked your Portfile with `port lint`?
- [x] tried a full install with `sudo port -vst install`?
- [x] tested basic functionality of all binary files?

### CODE REVIEW

**Comment by cjones051073** [comment]
An you check the build failure ?

**Comment by Tatsh** [comment]
The Azure pipeline recognises the `known_fail yes` [line](https://github.com/macports/macports-ports/blob/814b6fb0a1ed4d4d6e0414dbf94f1ac1ff37db2a/office/libreoffice/Portfile#L147) (and probably has statistically failed in the past primarily due to timeouts since the build takes a long time). Since the only changed subport is libreoffice, this message appears: *Excluding libreoffice because it is known to fail* ([link](https://dev.azure.com/macports/macports-ports/_build/results?buildId=15454&view=logs&j=ca395085-040a-526b-2ce8-bdc85f692774&t=731d6048-7790-5356-ea3a-50be86385f89&l=13)).

Then the Archive build logs and Publish build logs steps fail because there are no build logs.

File: office/libreoffice/Portfile [hunk 1]
```diff
@@ -155,15 +155,11 @@ if {${os.platform} eq "darwin" && ${os.major} < 19} {
...
-patchfiles \
-    patch-configure.diff \
-    patch-ucb_ucp_webdav_webdavprovider_cxx.diff \
-    patch-unpack-sources-fix-for-bsd-find.diff
-
 set product_name LibreOffice
 
 pre-configure {
     system "cd ${worksrcpath} && env NOCONFIGURE=1 ./autogen.sh"
+    reinplace "/# if the systemd-detect-virt command/d" Makefile.in
```
Comment (ryandesign) [inline office/libreoffice/Portfile:162]: I assume this reinplace is meant to be equivalent to this upstream commit?

https://github.com/LibreOffice/core/commit/cfab695d0e3bd5889cfa4b6e81300dd11d4eef72

Patchfiles are preferred over reinplaces. Can a patch be used instead? If Makefile.in is a generated file, can you patch the file it is generated from?

Comment (Tatsh) [reply office/libreoffice/Portfile:162]: Yes. I reported that issue on the mailing list. I will update this with that commit patch.

File: office/libreoffice/Portfile [hunk 2]
```diff
@@ -148,17 +148,14 @@ if {${os.platform} eq "darwin" && ${os.major} < 19} {
     pre-fetch {
         ui_error "Building ${name} @${version} requires macOS 10.15 or later."
         ui_error ""
-        ui_error "Consider downloading a version from LibreOffice's website:"
+        uieerror "Consider downloading a version from LibreOffice's website:"
```
Comment (ryandesign) [inline office/libreoffice/Portfile:151]:
```suggestion
        ui_error "Consider downloading a version from LibreOffice's website:"
```

**Pull request merged**